internal/resource/cuda-device.go

```diff
@@ -97,6 +97,18 @@ func (d *cudaDevice) IsMigEnabled() (bool, error) {
 	return false, nil
 }
 
+func (d *cudaDevice) IsImexCapable() (bool, error) {
+	return false, nil
+}
+
 func (d *cudaDevice) GetPCIClass() (uint32, error) {
 	return 0, nil
 }
+
+func (d *cudaDevice) GetClusterUUID() (string, error) {
+	return "", nil
+}
```

Comment:
I thought we wanted to return errors here?

Reply:
done